docs/development/deploying-at-scale/index.md

```diff
@@ -1,3 +1,41 @@
 # Deploying at scale
 
 REANA can be easily deployed on large Kubernetes clusters using Helm. Useful for production instances.
+
+## Pre-requisites
+
+REANA needs a shared file system to host all analyses workspaces when running in a multinode deployment setup. Therefore, you should create an [`StorageClass`](https://kubernetes.io/docs/concepts/storage/storage-classes/#the-storageclass-resource) pointing to your storage backend. The `StorageClass` should meet the following requirements:
+
+- Be named `<helm-release-prefix>-shared-volume-storage-class`
+- Be created in the same namespace as the one you will deploy REANA to
+
+[At CERN we use CephFS](https://clouddocs.web.cern.ch/containers/tutorials/cephfs.html) as storage backend.
+
+!!! note
+    You don't have a distributed file system? You can easily [deploy an NFS network file system following our documentation](../../advanced-usage/storage-backends/nfs).
```

Comment:
If you do not have any particular distributed file system in your Kubernetes cluster, you can easily...